components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc

```diff
@@ -104,6 +104,11 @@ _Always required_
 
 Initiates the execution stage in the sequencer. The sequencer calls this function upon receiving an `INVOKE` transaction, after the `+__validate__+` function successfully completes.
 
+[WARNING]
+====
+Deprecated `INVOKE` v0 transactions can call the `+__execute__+` function of all contracts. To mitigate this risk, `+__execute__+` calls should explicitly assert that the transaction version is 1 or above.
+====
+
```

Comment:
@stoobie not sure if the warning is in the correct format. Would also be good to link INVOKE v0 to this page: https://docs.starknet.io/architecture-and-concepts/network-architecture/transactions/#v0_deprecated_hash_calculation